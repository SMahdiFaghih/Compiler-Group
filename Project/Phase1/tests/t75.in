2int 2id 2inta 2aint 0x12int 0x12a