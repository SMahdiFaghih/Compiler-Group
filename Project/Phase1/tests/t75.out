T_INTLITERAL 2
int
T_INTLITERAL 2
T_ID id
T_INTLITERAL 2
T_ID inta
T_INTLITERAL 2
T_ID aint
T_INTLITERAL 0x12
int
T_INTLITERAL 0x12a